### ISSUE TITLE
Build an obvious reporting tool for PDF errors

### ISSUE THREAD

**Issue by hluedeke** [Pattern-Projector/pattern-projector#14]
It's expected that early in the development process, and with such a wide audience to test, that a lot of PDFs will throw errors.

We should build a popup or box that displays when a PDF fails with the error and instructions on how to report it (what is the process to report a bug? An email address?). This will hopefully increase reporting rate and visibility into potential errors.

**Comment by courtneypattison** [comment]
As I was troubleshooting a problem PDF last night, I was thinking same thing! I think email is probably the simplest. What do you think? I could just add any issues here when I get them.

**Comment by hluedeke** [comment]
The email is definitely simplest for starting out, and I think a lot of people will be supportive. There are fun tools like Rollbar that do automated issue management, but that also requires that we have an API setup (which I don't believe we have today, although I haven't really looked!)

I would create a new email address that isn't your personal one. Otherwise, I think it's great!

**Comment by courtneypattison** [comment]
Agreed! No API yet. I do have an email setup for PP. It's courtney@patternprojector.com. I should really link it up to my github account!

**Comment by hluedeke** [comment]
Ok, so I created a dialog that displays render and load errors, however, I'm a little concerned that it is too touchy. The dialog would display even when the pdf successfully loaded, though it's possible some of the pdf did not load correctly. I tried several PDFs tonight from several pattern companies, and almost all of them loaded without issue.

I will still add a PR, but we can review and decide whether this is the solution we want.
